client/components/domains/domain-search-results/index.jsx

```diff
@@ -148,7 +149,7 @@ var DomainSearchResults = React.createClass( {
 						domainsWithPlansOnly={ this.props.domainsWithPlansOnly }
 						railcarId={ `${ this.props.railcarSeed }-registration-suggestion-${ i }` }
 						uiPosition={ i }
-						fetchAlgo={ this.props.fetchAlgo }
+						fetchAlgo={ endsWith( suggestion.domain_name, 'wordpress.com' ) ? 'wpcom' : this.props.fetchAlgo }
```

Comment:
While `wordpress` is a blacklisted word and should not be present in the domain name, super admins are exempt from this restriction, so it is technically possible to register, for example, `somerandomwordpress.com`, which will get flagged as `wpcom`, when it's not. Changing `'wordpress.com'` to `'.wordpress.com'` should fix that :)